docs/src/pages/customization/components/components.md

```diff
@@ -5,90 +5,28 @@
 As components can be used in different contexts, there are several approaches to this. Going from the narrowest use-case to the broadest, these are:
 
 1. [Specific variation for a one-time situation](#1-specific-variation-for-a-one-time-situation)
```

Comment:
What do you think about simplifying the headers to make them easier to understand? (too much information to process otherwise)

```suggestion
1. [One-time customization](#1-one-time-customization)
```